FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -62,13 +63,15 @@ final class SetupInitialViewController: TableNodeViewController {
         backupService: BackupServiceType = BackupService(),
         router: GlobalRouterType = GlobalRouter(),
         decorator: SetupViewDecorator = SetupViewDecorator(),
-        organisationalRulesService: OrganisationalRulesServiceType = OrganisationalRulesService()
+        organisationalRulesService: OrganisationalRulesServiceType = OrganisationalRulesService(),
+        organisationalRulesPermissionsService: OrganisationalRulesPersmissionsServiceType = OrganisationalRulesPersmissionsService()
```

Comment:
I think it's not really permissions, more like an integrity check of the received OrgRules.

Can we call it ClientConfiguration instead?

`ClientConfigurationService`
`ClientConfigurationIntegrityCheckService` (although this would probably not need to be a separate service)